src/services/members.js

```diff
@@ -153,6 +153,17 @@ class MembersService {
     return getApiResponsePayloadV3(res);
   }
 
+  /**
+   * Deletes avatar for member.
+   * @param {String} userHandle The user handle
+   * @param {String} webLinkHandle The external web link handle
+   * @return {Promise} Resolves to the api response content
+   */
+  async deletePhoto(profile) {
+    const res = await this.private.api.delete(`/members/${profile.handle}/photo`);
```

Comment:
@Gauravseta does that work for you?

`DELETE /members/{handle}/photo` throws `405 Method Not Allowed` for me.

Reply:
Any update on this @Gauravseta ?

Reply:
@ThomasKranitsas i havent checked it, i just mimicked looking at other implementations and also by assuming correct response shall be receuved.
I feel Url is not proper. is the url correct? how can i come to know what url is to be hit?

Reply:
@ajefts @cwdcwd @urwithat thoughts on this?

Reply:
@ThomasKranitsas not to forget, everything for delete functionality is implmented in this repo, except the code which i have written above. this is done, then delete functionality shall work.